av/packet.pyx

```diff
@@ -204,6 +204,9 @@ cdef class Packet(Buffer):
             if self.struct.duration != lib.AV_NOPTS_VALUE:
                 return self.struct.duration
 
+    property flags:
```

Comment:
For new code, we prefer the modern form:

```
@property
def flags(self):
    return self.struct.flags
```

Reply:
Also, @mikeboers has some infrastructure in place for enums, would it be appropriate to use these?

My concern is that simply exposing "flags" as-is means that users will have to hard-code the AV_PKT_FLAG_xx in their code to interpret "flags".

Reply:
I use the flags for remux the stream, just copy it, so using the original flags value